mock/notifier/notifier.go

```diff
@@ -50,20 +49,6 @@ func (mr *MockNotifierMockRecorder) GetSenders() *gomock.Call {
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "GetSenders", reflect.TypeOf((*MockNotifier)(nil).GetSenders))
 }
 
-// InitImageStore mocks base method
-func (m *MockNotifier) InitImageStore(arg0 map[string]string) error {
-	m.ctrl.T.Helper()
-	ret := m.ctrl.Call(m, "InitImageStore", arg0)
-	ret0, _ := ret[0].(error)
-	return ret0
-}
-
-// InitImageStore indicates an expected call of InitImageStore
-func (mr *MockNotifierMockRecorder) InitImageStore(arg0 interface{}) *gomock.Call {
-	mr.mock.ctrl.T.Helper()
-	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "InitImageStore", reflect.TypeOf((*MockNotifier)(nil).InitImageStore), arg0)
-}
-
```

Comment:
Why this two methods was removed?

Reply:
I've rebuilded mocks with a script, and these methods appear to be no longer needed. At least, they are not in any interface in repository.

Reply:
Ok, I see. I believe that someone is removed methods from interface and forgot to rebuild mocks. I just thought that you generated mocks on branch different from master.